rt-shell/objects/obj_shell/Create_0.gml

```diff
@@ -195,6 +195,39 @@ function _find_common_prefix() {
 	return result;
 }
 
+function _string_split( _input, _delimiter )
```

Comment:
If it wouldn't be too much trouble I'd ask you to match the coding style of the repository (curly braces on the same line)!